### DISCUSSION TITLE
FR: allow remapping modifiers like iTerm2

### DISCUSSION THREAD

**Discussion by avamsi** [ghostty-org/ghostty#3331] [Feature Requests, Ideas] [answered]
Would like to map command as control on macOS.

**Comment by Me7426** [comment]
Yeah, want it too. This is a really important feature for me, as a Vim user :)

**Comment by mitchellh** [comment, marked as answer]
Thanks for posting this in the right place, but this is a duplicate. #3493

**Reply by avamsi** [reply]
Looks like #3376 / #3493 was filed later but I'll admit it probably does a better job describing, especially with the screenshot.. :shrug:

**Reply by mitchellh** [reply]
Yeah, first to post doesn't win, first that a maintainer looks at wins lol.
